### ISSUE TITLE
Critical Error: Plex cache raised an input/output error (disk I/O error)

### ISSUE THREAD

**Issue by werkaster** [trakt/Plex-Trakt-Scrobbler#419]
Windows 7 fully updated with latest Plex version. When I try to open trakt.tv in Plex to force sync I get the following message "Critical Error: Plex cache raised an input/output error (disk I/O error)" and nothing else.

Trakt.tv channel/scrobbler worked without problems for over a year I think, so this is very new. Obviously, it already says what went wrong, a disk I/O error, but all my HDDs work in Windows at the moment, and before I start running diagnostics on working drives, I thought I would check if there's an easy, trakt-scrobbler-specific, solution. Thank you.

**Comment by fuzeman** [comment]
You could try delete the `plex.db` file from:
```
C:\Users\<username>\AppData\Local\Plex Media Server\Plug-in Support\Caches\com.plexapp.plugins.trakttv
```

Sometimes these files can be corrupted for unknown reasons. Your next sync might take a bit longer as the cache will need to be rebuilt, but everything should return to normal after that.

**Comment by werkaster** [comment]
Thanks for the reply. I tried that and several other things, before and after asking here. Since then I've encountered two new errors, sometimes "This channel is not responding." and checking just now another one I've seen a few times "Critical Error: Plex cache is corrupt (database disk image is malformed)". 

Does this change your recommendation, or do you still think deleting caches\...…....trakttv\plex.db should be enough? I saw another .db-file in the folder if I remember correctly, but I didn't touch that one. As I said earlier, this could be an HDD problem rather than a plugin problem, but after several HDD scans with different software and not getting any alarming results, I'm a little confused.

**Comment by fuzeman** [comment]
Not sure what would cause the `This channel is not responding.` error, would need to see the plugin log file to identify that issue.

------

> Does this change your recommendation, or do you still think deleting caches...…....trakttv\plex.db should be enough? I saw another .db-file in the folder if I remember correctly, but I didn't touch that one.

I would try delete the `plex.db` file first as that's specifically what the error is mentioning. If the `Trakt cache is corrupt` error comes up as well, try delete the `trakt.db` file as well.

> As I said earlier, this could be an HDD problem rather than a plugin problem, but after several HDD scans with different software and not getting any alarming results, I'm a little confused.

An unclean process exit (system shutdown, or Plex Media Server shutdown) at the wrong time during a sync might of caused this as well, it's really hard to know what exactly caused this..

**Comment by LasseSLambertsen** [comment]
Any solution on this?
I'm getting almost the same error:
Critical Error: Trakt cache raised an input/output error

I've tried deleting everything related to trakt plugin, but no luck :/

**Comment by werkaster** [comment]
No, I'm still having problems, but to be honest, I haven't been doing everything I can to solve it, as nothing worked so far. I'm going to try again, and if I won't succeed I'll get some logs in here for fuzeman to look at. You should do the same as I've seen that message too.
